fetcher/src/job/job.utils.ts

```diff
@@ -1,11 +1,15 @@
 import { buildReducer, checkDataFormat, pipe, REDUCER_MAPPING } from '@bisonai/orakl-util'
 import { Logger } from '@nestjs/common'
 import axios from 'axios'
-import { FETCH_TIMEOUT } from '../settings'
+import { Contract, JsonRpcProvider } from 'ethers'
+import { abis as uniswapPoolAbis } from '../abis/pool'
+import { CYPRESS_PROVIDER_URL, FETCH_TIMEOUT } from '../settings'
 import { LOCAL_AGGREGATOR_FN } from './job.aggregator'
 import { FetcherError, FetcherErrorCode } from './job.errors'
 import { IAdapter, IFetchedData, IProxy } from './job.types'
 
+const CYPRESS_CHAIN_ID = 8217
```

Comment:
how about moving this statement to `../settings`

Reply:
Sure we can, but for now I think we're just dealing with 1 chain. We will have to change the whole thing to different chain definition method in the future.

Reply:
Can you elaborate a bit more about `change the whole thing to different chain definition method`? I couldn't get it..

Reply:
The best way is we have a table to define all popular chains, with name, IDs, RPCs and we load the chain setting from there. For now I can see we just dealing with 1 chain so I tried to keep it as simple as possible.

Reply:
I see, guess I was too skeptical about extending number of chains 😅